### PR TITLE
Local development

### PR DESCRIPTION
**Pull request by waTeim**
Added a Dockerfile and docker-compose file to support running a local helm repository that can take the part of serving up helm chats for **helx-charts**.  Updated the README for instructions on how to use it.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM node:latest
+RUN npm install -g http-server
+RUN mkdir /public
+WORKDIR /public
+
+EXPOSE 8080
+CMD ["http-server"]
```
Comment (mac-chaffee) [inline Dockerfile:7]: On my machine, this takes 67 seconds to build and results in a 1GB image :( Are there better alternatives?

One option is `python:3.10-alpine`, then use  `python -m http.server`. In fact, that python command will work on any computer with python3 installed (which should include all macos computers out-of-the-box), no containers needed.

**Comment by cschreep** [comment]
I'm not clear on the use case of this PR. What does it enable you to do?

**Comment by waTeim** [comment]
I'm not wed to any particular implementation of a web server, just a convenient-reliable one; for WS node > python, but for such a minimal case, I'll go with the flow and switch.  I'm opposed to not-docker.

The use case is development/testing without having to publish (it's in the README).

**Comment by cschreep** [comment]
The readme isn't clear on the justification. Why would someone need a webserver to test a helm chart?

**Comment by waTeim** [comment]
Ok, I can alter that to include something like enabling altering the contents helm-charts repo privately thus making the same commands work for both production and development, and helm allows repo definitions to only point to HTTP servers.

**Pull request merged**